src/analysis/normed_space/basic.lean

```diff
@@ -431,6 +465,12 @@ noncomputable def normed_space.of_core (α : Type*) (β : Type*)
 
 end normed_space
 
+section banach_space
```

Comment:
What is this doing in this PR? Do you intend to do more in the direction? I have a couple of lemmas somewhere in case you are interested in developing Banach spaces.